### ISSUE TITLE
the rendered face will be strange when the other 2 angle is large

### ISSUE THREAD

**Issue by HaoLiuHust** [fengju514/Face-Pose-Net#29]
the render only limit the yaw angle, is there a way to limit the other 2 angles?

**Comment by iacopomasi** [comment]
@HaoLiuHust Can you better explain what do you mean with to limit the two angles (tilt, roll)?

**Comment by HaoLiuHust** [comment]
I mean pitch and roll. right now I am limit these two angles before send them into main_fpn, makes the result seems better

**Comment by iacopomasi** [comment]
@HaoLiuHust Yes, this is reasonable. Though FPN is trained with strong in-plane data augmentation, it makes sense that if you remove some (other) pose variations in the input image, it will work better. Basically, you are feeding easier images to the system, I guess.
